### PR TITLE
Mdlint

### PR DESCRIPTION
**Pull request by obelic2**
- Added Markdown linter with changes
- Changed JSCS linter to accept spaces instead of tabs
Reference : Issue #159

### CODE REVIEW

File: .jscsrc-tools [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
 	"preset": "jquery",
 	"preset": "google",
-	"validateIndentation": "\t",
+	"validateIndentation": 2,
```
Comment (bbody) [inline .jscsrc-tools]: This is more likely a problem with your editor than the code. If it bothers you that much split into a different PR

Comment (obelic2) [reply .jscsrc-tools]: Yeah, I know Atom does soft tabs. spaces are just safer in general though

File: gulpfile.js [hunk 2]
```diff
@@ -136,299 +137,304 @@ gulp.task('test', ['watch', 'build', 'test:karma:build', 'coverage']);
...
 gulp.task('jsonlint', function() {
-	gulp.src(['**/*.json', '!node_modules/**'])
-		.pipe(jsonlint())
-		.pipe(jsonlint.report('verbose'));
+  gulp.src(['**/*.json', '!node_modules/**'])
+    .pipe(jsonlint())
+    .pipe(jsonlint.report('verbose'));
+});
+
+gulp.task('mdlint', function() {
```
[the remark this answers is not in the document]
Comment (obelic2) [reply gulpfile.js]: it should those flagged as "error"

File: .jscsrc-tools (hunk 1 above)
Comment (obelic2) [reply .jscsrc-tools]: I changed it back in a new commit rather than cherry-picking
